tests/foreman/installer/test_installer.py

```diff
@@ -1624,36 +402,29 @@ def test_positive_check_installer_hammer_ping(target_sat):
             assert 'ok' in line
 
 
-@pytest.mark.e2e
+@pytest.mark.stream
 @pytest.mark.upgrade
 @pytest.mark.tier3
-@pytest.mark.parametrize('filter', ['params', 'sections'])
-def test_installer_options_and_sections(filter):
-    """Look for changes on installer sections and options/flags
+@pytest.mark.build_sanity
```

Comment:
@jyejare I felt like this was a good candidate for the build sanity pipeline. Thoughts?

Reply:
Ok. I would like to what significant verification that this file will bring into sanity ? Also how likely this test is likely fail ? 

We are not worried about no additions of modules but we really care when its removed because it has impact on the major tests. So does this likely fails when new additions are introduced ?

Also, I see that the file we wanted to compare with would be removed going fwd making this tests fail. Even though we are not known when its going to happen is it good to have such tests in solidified sanity test suit ?

Thanks again for your due diligence about sanity checkup. Waiting for your answers.

Reply:
The main verification is that the modules we expected to land in a snap are actually there. Pretty unlikely that this would fail on a week to week basis. New additions are added for certain features, but it's not super common. If you'd rather wait to add this to the sanity build until we have a more final solution that is fine with me.

Reply:
Ok! Lets keep it for now and if we see a common failures in this test, we will take a decision at that time.